types/joi/index.d.ts

```diff
@@ -1,6 +1,13 @@
-// Type definitions for joi v10.4.2
+// Type definitions for joi v11.0.3
```

Comment:
since this is a major version upgrade, please add a new folder for the old version first. see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version